src/ng/templateRequest.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var $templateRequestMinErr = minErr('$compile');
+var $templateRequestMinErr = minErr('$templateRequest');
```

Comment:
Actually a lot of tests broke. 😨 I didn't think of implications of the change. I think we said that changing the error name/ namespace is a breaking change so we can't do that in 1.6.